warc-indexer/src/main/solr/solr/discovery/conf/schema.xml

```diff
@@ -89,28 +90,28 @@
         <field name="pdf_pdfa_is_valid" type="string" indexed="true" docValues="true" multiValued="false"/>
         <field name="postcode_district" type="string" indexed="true" docValues="true" multiValued="true"/>
         <field name="postcode" type="string" indexed="true" docValues="true" multiValued="true"/>
-        <field name="publication_date" type="tdate" indexed="true" stored="true" multiValued="false"/>
+        <field name="publication_date" type="date" indexed="true" stored="true" multiValued="false"/>
```

Comment:
Is this right? Why are we using `date` here and still using `tdate` above?

Reply:
Hmm... The difference between `date` and `tdate` is `precisionStep`, which is about granularity vs. performance. I don't have strong opinions here, so using `tdate` is fine by me.